EIPS/eip-draft_retrieval_domain.md

```diff
@@ -0,0 +1,116 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 5267
```

Reply:
![image](https://user-images.githubusercontent.com/45835846/179213439-9dd1f198-7b9d-4df6-9191-4bcd418a973f.png)
@SamWilsn how about having a custom error specifically when the EIP "number" is set to `<to be assigned>`?